app/models/feedback_reporters/feedback_reporter.rb

```diff
@@ -21,16 +22,8 @@ def description
   end
 
   def send_report!
-    HTTParty.post("#{ArchiveConfig.BUGS_SITE}/projects/#{project_id}/bugs",
-      headers: { 
-        "Content-Type" => "application/xml", 
-        "Accept" => "application/xml" 
-      },
-      basic_auth: {
-        username: ArchiveConfig.BUGS_USER,
-        password: ArchiveConfig.BUGS_PASSWORD
-      },
-      body: xml
+    HTTParty.post("#{ArchiveConfig.BUGS_SITE}",
+      body: "&xml=#{xml}"
```

Comment:
I did actually try that (since it was what was there before) but it doesn't work.